_data/faqs.yaml

```diff
@@ -1,76 +1,76 @@
+    By basing Hyku on Hyrax, which provides a simple and broad data model for works consisting of files, we are able to provide general support for any content type or file format. We also provide richer content type-specific behaviors for image-based works, and are articulating the requirements for those types now. Support for additional content types, including datasets, books, theses and dissertations, audiovisual media, and newspapers, are high priorities for future releases. Read the [March 2016 blog post](/2016/03/07/content_types_news.html) for more on this topic, and we will post more information about our content type analysis and plans in due course.
 - question: >-
-    What descriptive metadata standards will Hydra-in-a-Box support? Must my metadata be expressed in RDF?
+    What descriptive metadata elements and standards will Hyku support? 
   answer: >-
-    We have been gathering lots of input on metadata from a diverse set of potential users during the project's Design Phase. We have not yet determined the metadata requirements to be implemented when we first launch Hydra-in-a-Box, though it is likely that we will support the <a href="http://dp.la/about/map/">DPLA MAP v4</a> as default descriptive metadata for easier integration with DPLA hubs. As for crosswalking your metadata (which might be in XML) into a format that Hydra-in-a-Box will use (such as RDF), we're thinking about that, too, and when we have answers, we'll share them. In the meantime, we’d love to hear from you. Please share your suggestions using in our <a href="http://bit.ly/hydrainabox-ideas">Ideas form</a>. You may also <a href="mailto:hybox-contact@googlegroups.com">email us</a> with any questions or to talk to us in more detail about crosswalking/mapping strategies.
+    Hyku currently supports a basic metadata element set for a genetic work, the same set of Dublin Core elements that ships with Hyrax. We are augmenting this set in two significant ways. First, we are ensuring that it is compatible with the <a href="http://dp.la/about/map/">DPLA Metadata Application Profile v4</a> so that works in Hyku can be readily published for harvesting by DPLA and its hubs without further mapping. Second, as we extend support for specific content types in Hyku, we will add specific elements relevant and appropriate to those content types. <br/><br>As for crosswalking your metadata (which might be in XML) into a format that Hyku will ultimately use (such as RDF), we're actively talking to interested institutions in the Hydra community on migration and mapping tools to support this work. Updates and announcements about this work will be posted here. In the meantime, we’d love to hear from you about your metadata questions. Please share your suggestions using in our <a href="http://bit.ly/hydrainabox-ideas">Ideas form</a> or <a href="mailto:hybox-contact@googlegroups.com">email us</a> with any questions about crosswalking/mapping strategies.
```

Comment:
Typo: `Please share your suggestions using in our ` I believe the `in` is extraneous.